test/DocumentFragment.test.php

```diff
@@ -24,8 +30,8 @@ public function testQuerySelector() {
 	$this->assertSame($expectedFirstLi, $actualFirstLi);
 }
 
-public function testQuerySelectorAll() {
-	$document = new HTMLDocument();
+public function testQuerySelectorAllBeforeAddingToDocument() {
+    $document = new HTMLDocument(self::DOC_CONTENT_BEFORE_INSERT);
```

Comment:
How have some spaces made their way in here? Don't worry, I'll get the fixer automated soon.